src/components/org/apache/jmeter/visualizers/BSFListener.java

```diff
@@ -52,7 +52,7 @@ public void sampleOccurred(SampleEvent event) {
             mgr.declareBean("sampleResult", result, SampleResult.class);
             processFileOrScript(mgr);
         } catch (BSFException e) {
-            log.warn("Problem in BSF script "+e);
+            log.warn("Problem in BSF script. {}", e.toString());
```

Comment:
I wasn't able to determine which one is better myself without knowing the context. I left it with toString() for now, just assuming the original author thought stack trace was unnecessary.
By the way, I'd like to propose a new PR soon to have if condition for level checking in warn, info and debug level in case params invoking a method. I did that for debug level but want to extend the pattern in non-error level as well for better performance.